Formula/fceux.rb

```diff
@@ -36,7 +33,7 @@ def install
 
     # gdlib required for logo insertion, but headers are not detected
     # https://sourceforge.net/p/fceultra/bugs/756/
-    system "scons", "RELEASE=1", "GTK=0", "GTK3=1", "LOGO=0"
+    system "python2", "#{HOMEBREW_PREFIX}/bin/scons", "RELEASE=1", "GTK=0", "GTK3=1", "LOGO=0"
```

Comment:
We don't want it to use python 2, so this is not really a fix.

Reply:
Hi, could a patch making the scons buildscripts python3 compatible be acceptable to re-enable this formula?
https://github.com/Homebrew/formula-patches/pull/290

Reply:
@npinochet You should instead contribute that to upstream, to be integrated for the good of everyone, not just Homebrew.

Reply:
@gromgit I took the patch from upstream (https://github.com/TASVideos/fceux), but apparently they don't do releases since it's not the original repo (which hasn't been updated since 2016). Maybe a specific commit from the new repo can be used as source, but that doesn't seems right. Also It has changed a lot from the original repo and I couldn't compile it for macOS. So i'm not sure what would be the right way to fix this :P

Reply:
I think requesting a tag in the github repo instead is the better way. The current code is a lot behind in every aspect, not just python.

Reply:
I agree, I will try to to make a pr to the github repo fixing compilation on macOS and request a release tag. This pr can be closed now I guess.

Reply:
Will do, thanks for the effort.